packages/cli/src/commands/deploy.ts

```diff
@@ -175,6 +178,8 @@ export const deploy = new Command('deploy')
     const config = new ProjectConfig(opts.rootDir);
 
     try {
+      await telemetry.identify(opts.storeHash);
```

Comment:
We should add tests to make sure this is being called with the right storeHash. Same for any use case in the tests.